hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/qjournal/client/AsyncLoggerSet.java

```diff
@@ -206,197 +194,114 @@ List<AsyncLogger> getLoggersForTests() {
   ///////////////////////////////////////////////////////////////////////////
   
   public QuorumCall<AsyncLogger, GetJournalStateResponseProto> getJournalState() {
-    Map<AsyncLogger, ListenableFuture<GetJournalStateResponseProto>> calls =
-        Maps.newHashMap();
-    for (AsyncLogger logger : loggers) {
-      calls.put(logger, logger.getJournalState());
-    }
+    Map<AsyncLogger, ListenableFuture<GetJournalStateResponseProto>> calls = Maps.newHashMap();
+    loggers.forEach(l -> calls.put(l, l.getJournalState()));
     return QuorumCall.create(calls);    
   }
   
   public QuorumCall<AsyncLogger, Boolean> isFormatted() {
     Map<AsyncLogger, ListenableFuture<Boolean>> calls = Maps.newHashMap();
-    for (AsyncLogger logger : loggers) {
-      calls.put(logger, logger.isFormatted());
-    }
+    loggers.forEach(l -> calls.put(l, l.isFormatted()));
     return QuorumCall.create(calls);
   }
 
-  public QuorumCall<AsyncLogger,NewEpochResponseProto> newEpoch(
-      NamespaceInfo nsInfo,
-      long epoch) {
-    Map<AsyncLogger, ListenableFuture<NewEpochResponseProto>> calls =
-        Maps.newHashMap();
-    for (AsyncLogger logger : loggers) {
-      calls.put(logger, logger.newEpoch(epoch));
-    }
+  public QuorumCall<AsyncLogger,NewEpochResponseProto> newEpoch(long epoch) {
```

Comment:
public QuorumCall<AsyncLogger,NewEpochResponseProto> newEpoch(long epoch) {:32: ',' is not followed by whitespace. [WhitespaceAfter]